docs/how-to/debugging_env.rst

```diff
@@ -0,0 +1,88 @@
+.. list-table::
```

Comment:
I don't understand this page - it doesn't have a title and it's not in the TOC
![image](https://github.com/ROCm/HIP/assets/164940351/e98f8f0b-4e0d-432b-afa6-770b73256a7b)

Reply:
This is reused multiple times. How did you reach this page? I added to exclude files.